WordPressVIPMinimum/Sniffs/Filters/RestrictedFilterSniff.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * WordPressVIPMinimum Coding Standard.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ */
+
+namespace WordPressVIPMinimum\Sniffs\Filters;
+
+use PHP_CodeSniffer_File as File;
+use PHP_CodeSniffer_Tokens as Tokens;
+
+/**
+ * This sniff restricts usage of some filters
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ */
+class RestrictedFilterSniff implements \PHP_CodeSniffer_Sniff {
+
+	/**
+	 * List of restricted filter names.
+	 *
+	 * @var array
+	 */
+	public $restrictedFilters = array(
+		'upload_mimes' => true,
+	);
+
+	/**
+	 * Returns the token types that this sniff is interested in.
+	 *
+	 * @return array(int)
```

Comment:
It's fine to follow what [PHPCS does](https://github.com/squizlabs/PHP_CodeSniffer/blob/c056352b9c27a951775b526c01bf1223364df558/src/Util/Tokens.php#L550) for the type here: `array<int, int>`.

[Generics notation is supported in the draft PSR-5](https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#collections).